README.md

```diff
@@ -45,7 +45,7 @@ A short summary what the playbook actually does.
 
 ## License
 
-This work is licensed under a [Creative Commons Attribution-ShareAlike 4.0 International License](http://creativecommons.org/licenses/by-sa/4.0/).
```

Comment:
Besser auf https://creativecommons.org/licenses/by-sa/4.0/ setzen, da werden auch evt. juristische Schwachstellen gefixed und die Lizenz behält ihre Gültigkeit. Auch wenn das LICENSE file outdated ist, gilt weiterhin die verlinkte Version sofern vorhanden, das ist in der Lizenz so festgehalten. Unberührt bleiben jedoch größere Änderungen an der Lizenz, diese ändern die Versions Nummer.